ash-window/src/lib.rs

```diff
@@ -118,6 +118,59 @@ pub unsafe fn create_surface(
     }
 }
 
+/// Extensions necessary for creating a surface on windows.
+#[allow(dead_code)]
+const WINDOWS_SURFACE_EXTENSIONS: &'static [*const c_char] = &[
+    khr::Surface::name().as_ptr(),
+    khr::Win32Surface::name().as_ptr(),
+];
+/// Extensions necessary for creating a surface on unix.
+/// Note that this is not equal to the return value of [`enumerate_required_extensions`] on Unix due to the multiple window types.
+#[allow(dead_code)]
+const UNIX_SURFACE_EXTENSIONS: &'static [*const c_char] = &[
+    khr::Surface::name().as_ptr(),
+    khr::WaylandSurface::name().as_ptr(),
+    khr::XlibSurface::name().as_ptr(),
+    khr::XcbSurface::name().as_ptr(),
+];
+/// Extensions necessary for creating a surface on android.
+#[allow(dead_code)]
+const ANDROID_SURFACE_EXTENSIONS: &'static [*const c_char] = &[
+    khr::Surface::name().as_ptr(),
+    khr::AndroidSurface::name().as_ptr(),
+];
+/// Extensions necessary for creating a surface on macos.
+#[allow(dead_code)]
+const MACOS_SURFACE_EXTENSIONS: &'static [*const c_char] = &[
+    khr::Surface::name().as_ptr(),
+    ext::MetalSurface::name().as_ptr(),
+];
+/// Extensions necessary for creating a surface on ios.
+#[allow(dead_code)]
+const IOS_SURFACE_EXTENSIONS: &'static [*const c_char] = MACOS_SURFACE_EXTENSIONS;
+
+/// Extensions necessary for creating a surface on the current target platform.
+/// (Note that on Unix, this is not equal to the return value of [`enumerate_required_extensions`])
+pub const TARGET_EXTENSIONS: &'static [*const c_char] = {
```

Comment:
Since `TARGET` here refers to the current `target_os`, are we perhaps missing `_SURFACE_` in between? So:

```suggestion
pub const TARGET_SURFACE_EXTENSIONS: &'static [*const c_char] = {
```